sycl/source/detail/config.hpp

```diff
@@ -232,8 +232,10 @@ template <> class SYCLConfig<SYCL_PARALLEL_FOR_RANGE_ROUNDING_PARAMS> {
 };
 
 // Array is used by SYCL_DEVICE_ALLOWLIST and ONEAPI_DEVICE_SELECTOR.
+// The 'supportAcc' parameter is used by SYCL_DEVICE_ALLOWLIST which,
+// unlike ONEAPI_DEVICE_SELECTOR, also accepts 'acc' as a valid device type.
 const std::array<std::pair<std::string, info::device_type>, 6> &
-getSyclDeviceTypeMap();
+getSyclDeviceTypeMap(bool supportAcc = false);
```

Comment:
It looks like `= true` would be a better default and minimize changes. Or maybe change to `disableAcc`.

Reply:
Actually, I think @uditagarwal97 is right here.  Once the preview breaking changes stuff goes away, this will "just work".  If it defaults to true, that'll mean more code that has to be revisited.

Also, I hate variable names with negatives in them, like "disable". They are subtly confusing.  "enableX = true/false" reads well, but "disableX = false" requires the reader to think.

Reply:
Ok.